TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/ConditionallyReliableLoggingHttpRequestHandler.java

```diff
@@ -24,13 +23,11 @@ protected void channelFinishedReadingAnHttpMessage(ChannelHandlerContext ctx, Ob
             trafficOffloader.flushCommitAndResetStream(false).whenComplete((result, t) -> {
                 if (t != null) {
                     log.warn("Got error: " + t.getMessage());
-                    ctx.close();
-                } else {
-                    try {
-                        super.channelFinishedReadingAnHttpMessage(ctx, msg, httpRequest);
-                    } catch (Exception e) {
-                        throw new RuntimeException(e);
-                    }
+                }
```

Comment:
This should definitely be a policy that the user drives.  There should be at least two modes on offloader failures - proceed and block traffic.  We might want to get more granular by having different policies for mutable vs non-mutating calls too.

Reply:
Originally thought I would quickly add this behavioral policy in but think it requires a little more refinement. For now adding a comment in code and created a JIRA: https://opensearch.atlassian.net/browse/MIGRATIONS-1276

Reply:
Shouldn't you be closing the context?  This feels like it could be a bad resource leak.

Reply:
The problem I face is that if I close the context our pipeline ends, so it would be equivalent with saying if our Kafka offload fails we won't send the request to the source cluster.